### PR TITLE
[hi] Add content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

### LINKED ISSUE #30571: [hi] Localize intro-windows-in-kubernetes.md to Hindi
**This is a Feature Request**

**What would you like to be added**

We need to localize [intro-windows-in-kubernetes.md](https://github.com/kubernetes/website/blob/main/content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md) to Hindi.

**Comments**

The current working branch for `hi` localization work is `dev-1.22-hi.1`. Please make sure to open a PR against `dev-1.22-hi.1`.
For more info see Hindi localization progress tracking issue #29353 

/language hi
/kind feature

### PR DESCRIPTION
**Pull request by BudhirajaMadhav**
This PR adds the localisation for `content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md` in Hindi language. 

Co-Authored by: @AvineshTripathi 

Fixes #30571

### CODE REVIEW

File: content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: Windows containers in Kubernetes
+content_type: concept
+weight: 65
+---
+
+<!-- overview -->
+
+Windows एप्लिकेशन कई संगठनों में चलने वाली सेवाओं और अनुप्रयोगों के एक बड़े हिस्से का गठन करते हैं। [Windows कंटेनर](https://aka.ms/windowscontainers) प्रक्रियाओं और पैकेज निर्भरता को समाहित करने का एक तरीका प्रदान करते हैं, जिससे DevOps प्रथाओं का उपयोग करना और विंडोज अनुप्रयोगों के लिए क्लाउड नेटिव पैटर्न का पालन करना आसान हो जाता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:9]: > [Windows कंटेनर](https://aka.ms/windowscontainers) प्रक्रियाओं और पैकेज निर्भरता को समाहित करने का एक तरीका प्रदान करते हैं

Do we intend for this to be plural?

File: content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+title: Windows containers in Kubernetes
+content_type: concept
+weight: 65
+---
+
+<!-- overview -->
+
+Windows एप्लिकेशन कई संगठनों में चलने वाली सेवाओं और अनुप्रयोगों के एक बड़े हिस्से का गठन करते हैं। [Windows कंटेनर](https://aka.ms/windowscontainers) प्रक्रियाओं और पैकेज निर्भरता को समाहित करने का एक तरीका प्रदान करते हैं, जिससे DevOps प्रथाओं का उपयोग करना और विंडोज अनुप्रयोगों के लिए क्लाउड नेटिव पैटर्न का पालन करना आसान हो जाता है। 
+
+विंडोज-आधारित अनुप्रयोगों और लिनक्स-आधारित अनुप्रयोगों में निवेश वाले संगठनों को अपने कार्यभार को प्रबंधित करने के लिए अलग ऑर्केस्ट्रेटर की तलाश करने की आवश्यकता नहीं है, जिससे ऑपरेटिंग सिस्टम की परवाह किए बिना, उनके परिनियोजन में परिचालन क्षमता में वृद्धि होती है।
```
Comment (divya-mohan0209) [inline content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:11]: We either need to Windows or विंडोज across the blog as a standard. Having both will confuse folks.

Comment (divya-mohan0209) [reply content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:11]: Ditto with लिनक्स or Linux.

File: content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+<!-- body -->
+
+## कुबेरनेट्स में विंडोज नोड्स
```
Comment (divya-mohan0209) [inline content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:15]: We've mentioned Windows before. if there's a standard we're looking to adhere to, we need to conform to one.

File: content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+कुबेरनेट्स में विंडोज कंटेनरों के ऑर्केस्ट्रेशन को सक्षम करने के लिए, अपने मौजूदा लिनक्स क्लस्टर में विंडोज नोड्स शामिल करें। कुबेरनेट्स पर {{< glossary_tooltip text="पॉड्स" term_id="pod" >}} में विंडोज कंटेनरों को शेड्यूल करना लिनक्स-आधारित कंटेनरों को शेड्यूल करने के समान है।
```
Comment (divya-mohan0209) [inline content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:17]: > कुबेरनेट्स में विंडोज कंटेनरों के ऑर्केस्ट्रेशन को सक्षम करने के लिए

I am not sure if the plural of containers can be written as कंटेनरों . I believe ऑर्केस्ट्रेशन has a better word too.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Some initial nits

File: content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md (hunk 2 above)
Comment (budhirajamadhav) [reply content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:11]: Yup, I'm aware of this, actually the translator tool does this inconsistency. Will clear once and for all.

File: content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md (hunk 4 above)
Comment (budhirajamadhav) [reply content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:17]: > I am not sure if the plural of containers can be written as कंटेनरों .
Couldn't figure out a better alternative.

> I believe ऑर्केस्ट्रेशन has a better word too.
I've used it after confirming in the [slack channel](https://kubernetes.slack.com/archives/CJ14B9BDJ/p1637686690074900). The other suggestion was `आयोजन`, should we go for that?

File: content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md (hunk 1 above)
Comment (budhirajamadhav) [reply content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:9]: Can you please clarify, plural for what?

**Comment by budhirajamadhav** [comment]
FIxes #30571

File: content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,711 @@
...
+                   "10.96.0.0/12",   # Service subnet
+                   "10.127.130.0/24" # Management (host) subnet
+               ]
+   ```
+
+1. मेरा Windows नोड `नोडपोर्ट` प्रकार की सेवाओं तक नहीं पहुंच सकता है
+
+    नोड से स्थानीय नोडपोर्ट पहुंच स्वयं विफल हो जाती है। यह एक ज्ञात है सीमा। NodePort एक्सेस अन्य नोड्स या बाहरी क्लाइंट से काम करता है।
+    
+1. कंटेनर के vNICs और HNS एंडपॉइंट डिलीट जा रहे हैं
```
Comment (ashish-jaiswar) [inline content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:606]: replace डिलीट by हटाए

File: content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 6]
```diff
@@ -0,0 +1,711 @@
...
+    
+    यदि आप अभी भी समस्याओं का सामना कर रहे हैं, तो सबसे अधिक संभावना है कि आपका नेटवर्क कॉन्फ़िगरेशन[cni.conf](https://github.com/Microsoft/SDN/blob/master/Kubernetes/flannel/l2bridge/cni/config/cni.conf) कुछ अतिरिक्त ध्यान देने योग्य है। आप इस स्थिर फ़ाइल को कभी भी संपादित कर सकते हैं। NS कॉन्फ़िगरेशन अद्यतन किसी भी नए Kubernetes संसाधनों पर लागू होगा।
+             
+    Kubernetes नेटवर्किंग आवश्यकताओं में से एक (देखें [कुबेरनेट्स मॉडल](/docs/concepts/cluster-administration/networking/)) है क्लस्टर संचार के बिना होने के लिए आंतरिक रूप से एनएटी। इस आवश्यकता का सम्मान करने के लिए, एक है [अपवाद सूची](https://github.com/Microsoft/SDN/blob/master/Kubernetes/flannel/l2bridge/cni/config/cni.conf#L20) उन सभी संचारों के लिए जहां आप नहीं चाहते कि आउटबाउंड NAT हो। तथापि,इसका मतलब यह भी है कि आपको उस बाहरी आईपी को बाहर करना होगा जिसे आप क्वेरी करने का प्रयास कर रहे हैं `ExceptionList` से। तभी आपके Windows से आने वाला ट्रैफिक होगा पॉड्स को बाहरी दुनिया से प्रतिक्रिया प्राप्त करने के लिए सही ढंग से SNAT'ed किया जाना चाहिए। इसमें संबंध में, आपकी `ExceptionList` `cni.conf` में इस प्रकार दिखनी चाहिए:
```
[the remark this answers is not in the document]
Comment (BudhirajaMadhav) [reply content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:592]: That's actually indentation, its likewise in the English version

File: content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 7]
```diff
@@ -0,0 +1,711 @@
...
+* `podSecurityContext` (नीचे देखें)
+* `shareProcessNamespace` - यह एक बीटा सुविधा है, और यह Linux नेमस्पेस पर निर्भर करती है जो Windows पर लागू नहीं हैं। Windows प्रक्रिया नामस्थान साझा नहीं कर सकता या कंटेनर की रूट फाइल सिस्टम। केवल नेटवर्क साझा किया जा सकता है।
+* `terminationGracePeriodSeconds` - यह Windows पर डॉकर में पूरी तरह से लागू नहीं है, [गिटहब मुद्दा] (https://github.com/moby/moby/issues/25982) देखें। आज व्यवहार यह है कि ENTRYPOINT प्रक्रिया CTRL_SHUTDOWN_EVENT भेजी जाती है, तब Windows डिफ़ॉल्ट रूप से 5 सेकंड प्रतीक्षा करता है, और अंत में बंद हो जाता है सामान्य Windows शटडाउन व्यवहार का उपयोग करने वाली सभी प्रक्रियाएं। 5 दूसरा डिफ़ॉल्ट वास्तव में Windows रजिस्ट्री में है [कंटेनर के अंदर](https://github.com/moby/moby/issues/25982#issuecomment-42644183), इसलिए जब कंटेनर बनाया जाता है तो इसे ओवरराइड किया जा सकता है।
+* `volumeDevice` - यह एक बीटा सुविधा है, और इसे Windows पर लागू नहीं किया गया है। Windows कच्चे ब्लॉक डिवाइस को पॉड्स में संलग्न नहीं कर सकता है।
+* `volumes`
+  * यदि आप 'emptyDir' वॉल्यूम को परिभाषित करते हैं, तो आप इसके वॉल्यूम स्रोत को 'memory' पर सेट नहीं कर सकते हैं।
```
[the remark this answers is not in the document]
Comment (BudhirajaMadhav) [reply content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:437]: that is a different point under volumes

**Comment by BudhirajaMadhav** [comment]
Hey, @ashish-jaiswar I've fixed the above mentioned nits. Thanks for your review. Also, if you find another sentence that does not make sense(probably happened due to EOL before full stop) then do tell.

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]
Hey I noticed that this document contains a mixture of both `Kubernetes` and `कुबेरनेट्स`. We should stick to the hindi one I guess as like the other docs.

**Comment by BudhirajaMadhav** [comment]
Hey @anubha-v-ardhan @jayesh-srivastava I was away for some days due to my exams but I've fixed the localisation of `kubernetes` everywhere in the file where it was needed according to my knowledge. Also, I fixed some nits I found in @AvineshTripathi 's part. Let me know if there is something else that needs to be done.

File: content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md (hunk 1 above)
Comment (divya-mohan0209) [reply content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md]: The word container

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Kubernetes OR कुबेरनेट्स - please can we standardize across the document?

**Comment by jayesh-srivastava** [comment]
> Kubernetes OR कुबेरनेट्स - please can we standardize across the document?

@divya-mohan0209 I think we should use `कुबेरनेट्स` across all docs. Having the word 'Kubernetes' in english wouldn't make sense for hindi content.

**Comment by divya-mohan0209** [comment]
@jayesh-srivastava - I agree. @BudhirajaMadhav - Please can you check and address it? Kindly let us know if you're unable to continue work on this PR.

**Comment by divya-mohan0209** [comment]
Hey @BudhirajaMadhav , please could you comment if you'd be able to continue work on this PR? There has been no activity since January 2022.

**Comment by divya-mohan0209** [comment]
Hello @BudhirajaMadhav, I will be closing the PR since there has been no response from your end & no activity on this PR since Jan. Please reopen when you're comfortable working on the changes.
/close

**Pull request closed without merging**